### PR TITLE
Internal: Fetch API data inside the home module [ED-13983]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/home/api.php
Comment (davseve) [inline modules/home/api.php:6]: Is this supposed to be stg....red?

File: modules/home/api.php
Comment (davseve) [inline modules/home/api.php:8]: Why all the methods in this file are static?

Comment (hein-obox) [reply modules/home/api.php:8]: I have followed the implementation of the /notifications module as close as possible. Yakir had used all static methods. I assume he has done that to avoid additional instantiations of the class. I am happy to leave it as is, but 
I can change it as well. 

Another question: we possibly could refactor the code further and create a base for some of these methods. I just don't know where to create the base and how to name it. Wdyt?

Possibly we could do a huddle with you (and Louis?) just now if necessary.

File: modules/home/api.php
Comment (hein-obox) [reply modules/home/api.php:6]: I am waiting for this PR to be merged: https://github.com/elementor/editor-assets/pull/44. I can only update the URL to non-staging URL, once the other PR has been merged.

File: modules/home/api.php
Comment (hein-obox) [reply modules/home/api.php:8]: @davseve Maybe we can leave this question open, and check with @louiswol94 ?

File: modules/home/api.php
Comment (hein-obox) [reply modules/home/api.php]: Updated.

File: modules/home/assets/js/app.js
Comment (hein-obox) [inline modules/home/assets/js/app.js:21]: Small glitch.

File: includes/settings/settings.php
Comment (hein-obox) [inline includes/settings/settings.php:106]: We realized that WordPress doesn't support the dark mode. Therefore we shouldn't activate it on this page, because it will look odd.

File: modules/home/assets/js/app.js
Comment (hein-obox) [inline modules/home/assets/js/app.js:30]: We realized that WordPress doesn't support the dark mode. Therefore we shouldn't activate it on this page, because it will look odd.

File: modules/home/api.php
Comment (louiswol94) [reply modules/home/api.php:8]: I believe @hein-obox did very well to mimic the current behavior. But there is now a lot of duplicated code between apps/notifications/home-screen. Perhaps we can open a task under Engineering Excellence to refactor this code.

File: modules/home/module.php
Comment (louiswol94) [inline modules/home/module.php:33]: `enqueue_editor_scripts` - this is not an "editor" script.

Comment (hein-obox) [reply modules/home/module.php]: Updated.

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**
